### DISCUSSION TITLE
Add @TisVictress to Go Contributors

### DISCUSSION THREAD

**Discussion by TisVictress** [paketo-buildpacks/feedback#120] [Go Team]
Hey! I'm currently working on updating the BOM for `go-mod-vendor` to include go modules. I'd like to join the contributors team.

**Comment by fg-j** [comment]
@paketo-buildpacks/go-maintainers 👍 /👎  ?

**Comment by ForestEckhardt** [comment]
@TisVictress I would love to added a new Go contributor. I think that this contributions would be more than enough in my books to constitute becoming a contributor. That being said unless there is something blocking you from making this contribution, I would like see the work landed before promoting you to contributor.

**Comment by ryanmoran** [comment]
Congrats @TisVictress! You've been added to the Go Contributors team.
